bayesian/utils.py

```diff
@@ -1223,6 +1223,19 @@ def get_recommendation_feedback_by_ecosystem(ecosystem):
     }
 
 
+accepted_ecosystems = [
+    "npm",
+    "maven"
+]
+
+
+def check_for_accepted_ecosystem(ecosystem):
+    """Check if the ecosystem is supported or not."""
+    if ecosystem in accepted_ecosystems:
+        return True
+    return False
```

Comment:
just `return ecosystem in accepted_ecosystems` is sufficient

Reply:
done